contents/newsletter/remote-working.md

```diff
@@ -0,0 +1,97 @@
+
+Automattic’s Beau Lebens has written (of course) a [great guide](https://beau.blog/2020/03/remote-work-at-scale/) to the benefits of effective written communication with lots of practical tips. One key takeaway: tools are not the answer. Installing a bunch of bots to transcribe meetings and run automated functions in Slack will only work if you do all the hard work first. And nailing how you communicate is a big part of that. Writing things down is great because:
+
+* **It’s faster. **Communicating via writing, especially with remote staff in lots of different time-zones, saves a lot of time. Scheduling is hard and meetings interrupt deep work. If something can be written down, it will both save time and improve transparency around decision-making.
+* **It’s always there. **If a decision is made on a call and nobody writes it down, was a decision even made? Calls can be good, but calls are also information black holes. Transparency is crucial in remote-only companies. So, when you do have a meeting about something, make sure someone, [or something](https://www.sembly.ai/), is taking notes.
+* **It’s clearer. **Seriously, have you ever read a transcript of you speaking? Writing things down helps with intentionality as, hopefully, it makes you think through both what you’re trying to achieve and how you want to communicate it.
+
+### The problem: Meetings
+
+Online meetings are the scourge of remote work – and often a cause of tension between office-based and non-office based employees. Mics off when they should be on, screenshare chaos, [Jeffrey Toobin’s penis](https://www.wired.com/story/jeffrey-toobin-zoom-penis/) – they remain a technical nightmare and a productivity black hole. Many companies see them as a necessary evil – a way for colleagues, no matter where they’re based, to collaborate. This is wrong.
+
+Since the pandemic, the amount of time people have spent not just in online meetings, but in meetings in general, has soared. [Time spent in meetings](https://www.microsoft.com/en-us/worklab/work-trend-index/great-expectations-making-hybrid-work-work?ref=async.twist.com) has gone up 253%. Some Teams users [spend](https://www.microsoft.com/en-us/worklab/work-trend-index/will-ai-fix-work) close to eight hours a week, or an entire workday, in online meetings. Eight hours a week! In Teams! Madness. And if you’re a manager or executive within an organization, this is all your fault.
```

Comment:
```suggestion
Since the pandemic, the amount of time people have spent not just in online meetings, but in meetings in general, has soared. [Time spent in meetings](https://www.microsoft.com/en-us/worklab/work-trend-index/great-expectations-making-hybrid-work-work) has gone up 253%. Some Teams users [spend](https://www.microsoft.com/en-us/worklab/work-trend-index/will-ai-fix-work) close to eight hours a week, or an entire workday, in online meetings. Eight hours a week! In Teams! Madness. And if you’re a manager or executive within an organization, this is all your fault.
```